src/error_estimation/src/error_estimation_factory.C

```diff
@@ -108,8 +108,8 @@ namespace GRINS
 
           adjoint_error_estimator->qoi_set() = qoi_set;
 
-          adjoint_error_estimator->number_h_refinements = 1;
-          adjoint_error_estimator->number_p_refinements = 0;
+          adjoint_error_estimator->number_h_refinements = input( "MeshAdaptivity/arefee_h_refs", 1 );
```

Comment:
That's a horrible input option name. How about something like `n_adjoint_h_refinements`? Analogously for p version on the line below.